shard.yml

```diff
@@ -1,5 +1,5 @@
 name: placeos-compiler
-version: 4.5.0
+version: 4.5.1
```

Comment:
_Likely_ ok? You're across usage scenarios a little more than me. Is there anywhere this change could cause an incompatibility worthy of a minor bump?